src/main/java/at/fhhagenberg/service/RMIElevatorService.java

```diff
@@ -4,23 +4,43 @@
  */
 package at.fhhagenberg.service;
 
-import java.rmi.RemoteException;
+import java.rmi.*;
 
 import at.fhhagenberg.logging.Logging;
 import sqelevator.IElevator;
 
 public class RMIElevatorService implements IElevatorService{
-    private final IElevator mApi;
+    private IElevator mApi;
+
+    private final String mConnectionString;
 
     private static final String ERROR_MESSAGE_LITERAL = "! \nError message: ";
 
-    public RMIElevatorService(IElevator api) {
-        if (api == null) {
-            String errorMsg = "Could not create RMIElevatorService, the given IElevator object is null!";
+    public RMIElevatorService(IElevator api, String connectionString) {
```

Comment:
Was nützt der API-Parameter

Reply:
der is fürs testing zum wegmocken von da API

Reply:
kunnt ma üba an setter theoretisch ah lösen